klog.go

```diff
@@ -1313,6 +1365,29 @@ func Errorf(format string, args ...interface{}) {
 	logging.printf(errorLog, logging.logr, format, args...)
 }
 
+// ErrorS structured logs to the ERROR, WARNING, and INFO logs.
+// the err argument used as "err" field of log line.
+// The msg argument used to add constant description to the log line.
+// The key/value pairs would be join by "=" ; a newline is always appended.
+//
+// Basic examples:
+// >> klog.ErrorS(err, "Failed to update pod status")
+// output:
+// >> E1025 00:15:15.525108       1 controller_utils.go:114] "Failed to update pod status" err="timeout"
+func ErrorS(err error, msg string, keysAndValues ...interface{}) {
+	if logging.logr != nil {
+		logging.logr.Error(err, msg, keysAndValues)
+		return
+	}
+	b := &bytes.Buffer{}
+	b.WriteString(fmt.Sprintf("%q", msg))
+	b.WriteByte(' ')
+	b.WriteString(fmt.Sprintf("err=%q", err.Error()))
```

Comment:
Could you just call infoS (renamed to `printS`) with error as argument?

Reply:
done